spec/features/sidebar_spec.rb

```diff
@@ -8,4 +8,14 @@
 
     expect(active_link.text).to eq "Customers"
   end
+
+  it "displays translated name of model" do
+    allow(Customer.model_name).to receive(:human).and_return('Users')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.